src/table/components/TableWrapper.jsx

```diff
@@ -123,7 +123,7 @@ export default function TableWrapper(props) {
   };
 
   const tableContainerStyle = {
-    height: constraints.active || footerContainer ? '100%' : 'calc(100% - 52px)',
+    height: constraints.active || footerContainer || totalVerticalCount <= 10 ? '100%' : 'calc(100% - 52px)',
```

Comment:
I would do something like `const noPaginationNeeded = totalVerticalCount <= 10` since you use it in three different places

Reply:
Right. Thanks. Updated.